src/main/java/org/openstreetmap/atlas/checks/distributed/IntegrityCheckSparkJob.java

```diff
@@ -403,9 +404,11 @@ protected SwitchList switches()
      *            set of {@link BaseCheck}s to execute
      * @return {@code true} if sanity check passes, {@code false} otherwise
      */
-    private boolean isValidInput(final StringList countries, final Set<BaseCheck> checksToExecute)
+    private boolean isValidInput(final StringList countries,
+            final Map<String, Set<BaseCheck>> checksToExecute)
     {
-        if (countries.size() == 0 || checksToExecute.size() == 0)
+        if (countries.size() == 0
+                || checksToExecute.values().stream().anyMatch(checks -> checks.size() == 0))
```

Comment:
I'd suggest avoiding the bulk load operation. That would be hard to debug. Consider loading configuration without any overrides and checking if that is valid or not in this method. Then, per country read in group/country specific configuration and use that for execution.